R/carbon_analysis.R

```diff
@@ -0,0 +1,72 @@
+## pull in data for carbon lakes analysis
```

Comment:
Whoops, forgot my PR for the figure wasn't merged before pushing this. 

@ldecicco-USGS what do you think? Am I in the ballpark? Collated the groups of names using http://www.waterqualitydata.us/Codes/Characteristicname?mimeType=xml 

Need some help catching errors with the server 
```r
> all_P_data <- lapply(stateCd$STUSAB[c(31, 40)], queryByState, characteristicNames = DIC_names)
Error in curl::curl_fetch_memory(url, handle = handle) : 
  Server returned nothing (no headers, no data) 

# BUT works when I run them separately
all_DIC_list <- lapply(stateCd$STUSAB[31], queryByState, characteristicNames = DIC_names)
all_DIC_list2 <- lapply(stateCd$STUSAB[40], queryByState, characteristicNames = DIC_names)
```